councilmatic/settings.py

```diff
@@ -32,8 +32,10 @@
     AWS_S3_ACCESS_KEY_ID=(str, ""),
     AWS_S3_SECRET_ACCESS_KEY=(str, ""),
     AWS_STORAGE_BUCKET_NAME=(str, ""),
-    RECAPTCHA_PUBLIC_KEY=(str, ""),
-    RECAPTCHA_PRIVATE_KEY=(str, ""),
+    # Test keys from:
+    # https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do
+    RECAPTCHA_PUBLIC_KEY=(str, "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"),
```

Comment:
Locally, I had to remove the corresponding (empty) recaptcha variables from `.env.local` to get this to work. `django-environ` seems to parse empty variables as present and won't use the default. 

We should probably remove the these variables from `.env.local.example`.

Reply:
Great catch. Done!